atlas/responder/pkg/svc/zserver_test.go

```diff
@@ -0,0 +1,35 @@
+package svc
+
+import (
+	"testing"
+)
+
+func TestGetVersion(t *testing.T) {
```

Comment:
I think this code is useless and should be removed

Reply:
Done